src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  todo: Todo;
+  unsetSelectedTodo: () => void,
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, unsetSelectedTodo }) => {
+  const [user, setUser] = useState({
+    id: 0,
```

Comment:
I recommend to move it to variable even though it's used 1 time as it just makes code cleaner and easier to understand what kind of information it is.